MantidQt/RefDetectorViewer/src/RefMatrixWSImageView.cpp

```diff
@@ -69,5 +69,4 @@ RefIVConnections *RefMatrixWSImageView::getConnections() {
 
 RefMatrixWSImageView::~RefMatrixWSImageView() {
   // nothing to do here, since m_imageView is deleted when the window closes
-  delete m_imageView;
```

Comment:
This was an accidental commit, the code wasn't previously there and was added by accident by the parent commit e5bde51. 

However looking at the constructor code, I don't think `m_imageView` is ever deleted, because it has no parent, so this could be a possible memory leak?